cmd/mdatagen/metadata-schema.yaml

```diff
@@ -1,6 +1,17 @@
-# Required: type of the receiver.
+# Required: The type of the component.
 type:
 
+# Required: A high-level view of the development status and use of this component
+status:
+  # Required: The class of the component (For example receiver)
+  class: <receiver|processor|exporter|cmd|connector|extension>
```

Comment:
`cmd` is not a valid component.

Reply:
Removed

Reply:
@atoulme I just realized the `telemetrygen` command makes use of the metadata generator so we may need to add `cmd` as a valid class.
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/a33b9021422a000ed3feeaa8d522c31f055e7c34/cmd/telemetrygen/metadata.yaml#L1-L7

Reply:
@codeboten you added this, can I get your view on what we should do with `cmd` as a class value?

Reply:
Hmmm... it's a good question. I added it because i wanted to be able to generate the data for tools as well, but maybe its not necessary